lib/pages/editor/editor.dart

```diff
@@ -890,31 +888,37 @@ class EditorState extends State<Editor> {
     );
     final thumbnailSize = Size(720, 720 * previewHeight / page.size.width);
     final thumbnail = await screenshotter.captureFromWidget(
-      Theme(
-        data: ThemeData(
-          brightness: Brightness.light,
-          colorScheme: const ColorScheme.light(
-            primary: EditorExporter.primaryColor,
-            secondary: EditorExporter.secondaryColor,
```

Comment:
Changes in createThumbnailPreview  cause exceptions. Previous version of this function does not cause exceptions.

Please look at this code changes @adil192

Reply:
These changes were needed to not use the editor's `context` when taking a screenshot. Without this, taking a screenshot in the dispose method fails

Reply:
I cannot help with this I am not very familiar with widgets.